app/views/promotion/weeklyLandingPage.scala.html

```diff
@@ -38,7 +39,8 @@
         title = s"$title | The Guardian",
         bodyClasses = List("is-wide"),
         product = Some(WeeklyZoneA),// don't specifically need zone a, any weekly is fine
-        hreflangs = Some(hreflangs)
+        hreflangs = Some(hreflangs),
+        edition = DigitalEdition.getForCountry(Some(country))
```

Comment:
I'll do that in another PR as it causes changes in 26 files, and I don't want to distract from this change's intent. https://trello.com/c/yPaaPWG3/201-rename-digitaledition-to-guardianedition